spec/report_spec.rb

```diff
@@ -147,16 +147,9 @@
   end
 
   describe '.gregorian_calendar' do
-    context 'when given time is end of day' do
```

Comment:
hmm, we used to care about time (hour / min / sec) for date range for adtech api but that caused date range confusion / timezone issue, so hard coded to `00:00:00` for all date ranges, it seems adtech api don't care time for date ranges.